### DISCUSSION TITLE
Lock account after biometrics reset

### DISCUSSION THREAD

**Discussion by Prolonz** [ente-io/ente#1042] [Enhancements]
As has been recently reported, if a bad actor was able to watch a passcode being entered on iOS or Android before they stole the device they can currently change the biometrics and gain access to ente Authenticator even if it’s secured with biometrics. 

It would be great there was an option to automatically log out ente Authenticator anytime the biometrics changed to mitigate the risk of this kind of attack. Bitwarden already does this. 

Thanks devs for such a great app!

**Comment by mrbluecoat** [comment]
Also might be nice to state "Biometrics" explicitly on the Settings > Security list rather than just the current `Lockscreen` option.

**Comment by kaizokan** [comment]
Nice suggestion. I like to add a similar suggestion: Auto logout #7699 after a period of time. Someone who has access to the device PIN (or to the person/biometrics) can already access Ente Auth without having to change security settings. Having your account logged out when not in use prevents this.

**Comment by pejfat08-rgb** [comment]
__**
**__

**Comment by pejfat08-rgb** [comment]
,,,,,,, '.
